geoapps/utils/plotting.py

```diff
@@ -580,17 +581,11 @@ def input_string_2_float(input_string):
     """
     Function to input interval and value as string to a list of floats.
 
-    Parameter
-    ---------
-    input_string: str
-        Input string value of type `val1:val2:ii` and/or a list of values `val3, val4`
+    :param input_string: Input string value of type `val1:val2:ii` and/or
```

Comment:
can have typing in signature if not in doc anymore
```
def input_string_2_float(input_string: str) -> npt.NDArray[np.int_] :
```

See typing annotations for numpy: https://numpy.org/devdocs/reference/typing.html#api